examples/quantumgames/example_nonlocal_game.py

```diff
@@ -0,0 +1,489 @@
+"""Nonlocal games
+================================================================
+
+In this tutorial, we are going to cover the notion of a *nonlocal game*; a
+mathematical framework that abstractly models a physical system. The simplest
+instance of a nonlocal game involves two players, Alice and Bob, who are not
+allowed to communicate with each other once the game has started and who play
+cooperatively against an adversary referred to as the referee.
+"""
```

Comment:
You could use one of the images in this tutorial as a thumbnail. There is a default thumbnail for those that do not have a specified thumbnail.